conans/test/unittests/client/generators/cmake_test.py

```diff
@@ -340,8 +340,7 @@ def apple_frameworks_test(self):
                       '"path/to/Frameworks1"\n\t\t\t"path/to/Frameworks2")', content)
         self.assertIn('find_library(CONAN_FRAMEWORK_OPENCL OpenCL PATHS '
                       '"path/to/Frameworks1"\n\t\t\t"path/to/Frameworks2")', content)
-        self.assertIn('set(CONAN_LIBS_MYPKG  ${CONAN_FRAMEWORK_OPENGL} '
-                      '${CONAN_FRAMEWORK_OPENCL})', content)
+        self.assertIn('set(CONAN_LIBS_MYPKG )', content)  #FIXME: Add a CONAN_FRAMEWORKS_MYPKG
```

Comment:
This change was made in order to be able to differentiate libs from frameworks from system_libs. Otherwise, everything is mixed and you can't properly create the CMake targets.  Part of the reason I created #5909

Reply:
It is totally possible to differentiate, I think it is better to be consistent and not break because of implementation reasons.